qa-kidsfirst.planx-pla.net/manifests/fence/fence-public-config.yaml

```diff
@@ -0,0 +1,146 @@
+APP_NAME: Gen3 Data Commons
```

Comment:
Please add `ENABLE_DB_MIGRATION: false`

Reply:
i think the fence version doesn't know this variable yet, but when we update it will!